hmcts/pre-api
# S28 3152/cleanup streaming locators

### JIRA ticket(s)

- S28-3152

### Change description

- Cron job to delete all streaming locators and content key policies

> [!IMPORTANT]
> This PR requires manual testing by QA. Please notify the QA team @hmcts/pre-rec-evidence-qa.

**QA instructions**

- Create some streaming locators and content key policies (can be done through starting to watch a vod recording)
- Ask me to run this task
- Observe the streaming locators and content key policies have all been deleted for MK/AMS depending on which is being tested.
